test/javacVMOptions/Test.java

```diff
@@ -25,6 +25,7 @@
  * @test
  * @compile Test.java
  * @compile -J-DmyProperty=123 -processor Test -proc:only Test.java
+ * @compile Test.java -XDrawDiagnostic
```

Comment:
Please, also insert a line reading ` * @bug 7902510 7904002` under the ` * @test` line.

And update the copyright years to read: `* Copyright (c) 2019, 2025, Oracle and/or its affiliates. All rights reserved.`

Reply:
Sure!

Would it be better to add a separate test case for it instead of just adding another `@compile` line to this one?

Reply:
Doesn't have to be a dedicated test per issue; as long as the issue numbers are recorded via the `@bug` tag. Bonus points for adding `@comment` lines.